src/include/btmem.h

```diff
@@ -489,10 +489,23 @@ struct __wt_page_modify {
 	WT_SPINLOCK page_lock;		/* Page's spinlock */
 
 	/*
-	 * The write generation is incremented when a page is modified, a page
-	 * is clean if the write generation is 0.
+	 * The page state is incremented when a page is modified.
+	 *
+	 * WT_PAGE_CLEAN --
+	 *	The page is clean.
+	 * WT_PAGE_DIRTY_FIRST --
+	 *	The page has 1 unreconciled update.
```

Comment:
I'd prefer `The page is in this state after the first operation that marks a page dirty, or when reconciliation checking to see if it's done enough work to be able to mark the page clean`.

Reply:
4c3cd34

I removed the block comment below it since it was effectively saying the same thing.